lib/elixir/test/elixir/exception_test.exs

```diff
@@ -259,8 +259,8 @@ defmodule ExceptionTest do
       :exit, reason -> reason
     end
 
-    assert Exception.format_exit(reason) ==
-           "exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process"
+    expected_to_start_with = "exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process:"
+    assert Exception.format_exit(reason) |> String.starts_with?(expected_to_start_with)
```

Comment:
Probably we can use `=~` instead of `String.starts_with?/1`, I think we don't need to be that strict.